.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
 before_install:
   - gem update --system 2.5.1
   - gem --version
+  - gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
```

Comment:
@redconfetti , this has nothing to do with postgres, correct? what is this for?

Reply:
I had to have some sort of file edit to make a PR that would trigger a Travis CI build, so I chose the README.md, which needed to be updated.

See https://docs.travis-ci.com/user/languages/ruby/#bundler-20

This isn't working, so now I'm trying to see if specifying the JDK version will change the error output.